### PR TITLE
added resetdb.py

### PR DESCRIPTION
**Pull request by ArmandDj**
resetdb: run to flush database and enter test data
to run:
- python manage.py shell
- execfile('resetdb.py')

difficulty: added difficulty field to code model, difficulty is updated by average score and number of code smells

### CODE REVIEW

**Comment by hannakohxt** [comment]
Is there a default difficulty?  Also why are you taking the max of 100 or the number of original code smells \* 10? 

I think you need to add this line after setting the difficulty: "code.save()", but I could be wrong.

**Comment by ArmandDj** [comment]
new database model: Smell
resetdb.py populates Smells from codesmells.txt
difficulty fixes

File: codesniff/app/views.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def post(self, request, *args, **kwargs):
     	data = request.data
         user = data['creator']
     	code = data['code']
-    	code = Code(title=code['title'], content=code['content'], language=code['language'], creator_id=user)
+    	code = Code(title=code['title'], content=code['content'], language=code['language'], creator_id=user, difficulty=min(len(smells)*10, 100))
```
Comment (hannakohxt) [inline codesniff/app/views.py:145]: Where are you defining smells in len(smells)?

**Comment by ArmandDj** [comment]
fixed, sorry about that

File: codesniff/app/views.py (hunk 1 above)
Comment (ArmandDj) [inline codesniff/app/views.py:145]: fixed, sorry about that

**Pull request merged**